class LogAnalysisError(Exception):
    """自定义日志分析异常"""
    pass